v2/features/swap/useTokenList.ts

```diff
@@ -0,0 +1,25 @@
+import DEFAULT_TOKEN_LIST from "@uniswap/default-token-list";
+import { TokenInfo } from "@uniswap/token-lists";
+
+export type TokenAddressMap = {
+  [chainId: number]: {
+    [tokenName: string]: {
+      token: TokenInfo;
+    };
+  };
+};
+
+export const getListOfTokens = () => {
+  return DEFAULT_TOKEN_LIST.tokens.reduce<TokenAddressMap>((acc, prev) => {
```

Comment:
If we work with this list, please make sure the display names are right:

<img width="649" alt="Screen Shot 2022-05-22 at 18 09 27" src="https://user-images.githubusercontent.com/7811733/169692253-ef0dfeb8-875a-495a-bb10-c1716f73f61a.png">